src/index.js

```diff
@@ -1,11 +1,11 @@
-const { Telegraf } = require('telegraf');
-require('dotenv').config();
+const { Telegraf } = require("telegraf");
```

Comment:
**nitpick**: Consider using single quotes for consistency with common JavaScript style guides unless double quotes are required by project standards.